client_apis/python/src/tests/integration/alerts_test.py

```diff
@@ -0,0 +1,212 @@
+#
+# CARBON BLACK API TESTS - alerts
+# Copyright, Bit9, Inc 2015
+#
+
+""" These tests require CarbonBlack Enterprise Server to be installed.
+    You can run this script by passing the server URL and user API token
+    as parameters.
+"""
+
+import unittest
+import sys
+import os
+import json
+import requests
+import time
+import paramiko, base64
+
+
+if __name__ == '__main__':
+    sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../")))
+
+from cbapi.cbapi import CbApi
+from datetime import datetime
+
+cb = None
+
+class ShhHelper:
+    @classmethod
+    def execute_command(cls, server, user, password, command):
+        client = paramiko.SSHClient()
+        client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        client.connect(server, username=user, password=password)
+
+        transport = client.get_transport()
+        session = transport.open_session()
+        session.set_combine_stderr(True)
+        session.get_pty()
+
+        session.exec_command("sudo -k %s" % command)
```

Comment:
Will sudo always be required for all commands that you'd issue via this helper?  I'm thinking that you might want this to be optional